changelog/964.misc.rst

```diff
@@ -0,0 +1 @@
+do not load entrypoint plugins during tests by default with ``newconfig`` fixture - by :user:`blueyed` (`#964 <https://github.com/tox-dev/tox/pull/964>`_)
```

Comment:
please remove the issue reference, this is automatically added. hence why the city broke